e2e/specs/activeVisits.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { Encounter } from './../../packages/esm-active-visits-app/src/visits-summary/visit.resource';
+import { test } from '../core';
+import { HomePage } from '../pages';
+import { expect } from '@playwright/test';
+import { createEncounter, deletePatient, endVisit, generateRandomPatient, Patient, startVisit } from '../commands';
+import { Visit } from '@openmrs/esm-framework';
+
+let patient: Patient;
+let visit: Visit;
+let encounter: Encounter;
+const encounterNote = 'This is a test note';
+
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
+  visit = await startVisit(api, patient.uuid);
+  encounter = await createEncounter(api, patient.uuid, encounterNote);
+});
+
+test('should be able to see the active visits', async ({ page }) => {
+  const homePage = new HomePage(page);
+  await homePage.goto();
+
+  await homePage.clickOnActiveVisitPatient(patient.uuid);
+
+  // Checks for the encounter
+  await expect(
+    page.getByTestId('encountersTable').getByRole('cell', { name: encounter.encounterType.display }),
+  ).toBeTruthy();
```

Comment:
Can't we check for a value?

Reply:
then each cell require unique id. This is pretty much the same. we are checking the created value to exist.